src/main/kotlin/no/nav/medlemskap/regler/common/Personfakta.kt

```diff
@@ -51,11 +51,15 @@ class Personfakta(private val datagrunnlag: Datagrunnlag) {
     }
 
     fun sisteArbeidsforholdSkipsregister(): List<String> {
-        return hentArbeidsforholdIPeriode().flatMap { it -> it.arbeidsavtaler.map { it.skipsregister?.name.toString()} }
+        return hentArbeidsforholdIPeriode().flatMap { it.arbeidsavtaler.map { it.skipsregister?.name.toString()} }
     }
 
     fun hentBrukerinputArbeidUtenforNorge(): Boolean = datagrunnlag.brukerinput.arbeidUtenforNorge
 
+    fun hentTotalStillingprosenter(): List<Double> {
```

Comment:
Er dette en for enkel sjekk? Hvis jeg skjønner dette riktig, så lager du her en liste over alle stillingsprosentene for alle arbeidsforholdene som vi anser som relevante for perioden (mao som overlaps eller encloses), og så summerer du dem og sjekker dem mot en grenseverdi. Men da har vi jo ikke vurdert hvor lenge et arbeidsforhold varte? Hvis jeg har hatt en jobb i en uke som var 100%, så vil summen av stillingsprosentene bli 100. Men jeg har jo egentlig reelt sett jobbet bare bittelittegrann..?

Reply:
Mulig det, jeg gikk foreløpig ut i fra stillingsprosentene som lå i datagrunnlaget, men det trur jeg også må defineres mer fra Helle. Jeg tenker jo også at man kanskje bør sjekke opp mot hvor mye noen har tjent, klarer ikke helt å se felt i aareg hvor mye en person faktisk har jobbet.